e2e/rook-test-framework/clients/k8s_rook_block.go

```diff
@@ -0,0 +1,138 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+)
+
+type k8sRookBlock struct {
+	transportClient contracts.ITransportClient
+}
+
+var (
+	listCmd              = []string{"rook", "block", "ls"}
```

Comment:
We are trying to run end to end integration test - i.e. all tests are outside the k8s cluster, it basically.Can we use the client library from outside the cluster?
Ideally, we would like to hit the rook-api do all the set up needed, but you can only access rook api from the rook namespace withing the cluster.  In future iterations, we would like to figure out how to use rook-ai or client libraries in future iterations.